Custom-Pylint-checks.md

```diff
@@ -499,11 +499,11 @@ def test_finds_hello_world_assignment(self):
     temp_file = tempfile.NamedTemporaryFile()
     filename = temp_file.name
 
-    with python_utils.open_file(filename, 'w') as tmp:
-        tmp.write("""
-        s = "Hello, world!"
-        print("Hello, world!")
-        """)
+    with open(filename, 'w') as tmp:
+            tmp.write("""
```

Comment:
Deindent this and the following lines by 4.

Reply:
done